ksp_plugin_adapter/flight_planner.cs

```diff
@@ -413,13 +440,12 @@ class FlightPlanner : SupervisedWindowRenderer {
     return true;
   }
 
-  // Not owned.
```

Comment:
That comment should have been on `plugin_`, but it still applies.

Reply:
Done.